### ISSUE TITLE
Could not launch on macOS 10.15

### ISSUE THREAD

**Issue by ryandesign** [unsound/hfsexplorer#35]
On macOS 10.15 I downloaded hfsexplorer-2021.10.9-bin.zip and tried to run `./hfsexplorer` from within the bin directory. The Dock icon appeared briefly, then disappeared, and this error was shown in the terminal:

```
$ ./hfsexplorer
System L&F class name: com.apple.laf.AquaLookAndFeel
Exception in thread "main" java.lang.RuntimeException: java.lang.ClassNotFoundException: com.apple.eawt.ApplicationListener
	at org.catacombae.hfsexplorer.MacUtil.registerMacApplicationHandler(MacUtil.java:83)
	at org.catacombae.hfsexplorer.FileSystemBrowserWindow.<init>(FileSystemBrowserWindow.java:167)
	at org.catacombae.hfsexplorer.FileSystemBrowserWindow.<init>(FileSystemBrowserWindow.java:154)
	at org.catacombae.hfsexplorer.FileSystemBrowserWindow.openFileSystemBrowserWindow(FileSystemBrowserWindow.java:3252)
	at org.catacombae.hfsexplorer.FileSystemBrowserWindow.main(FileSystemBrowserWindow.java:3304)
Caused by: java.lang.ClassNotFoundException: com.apple.eawt.ApplicationListener
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:581)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
	at java.base/java.lang.Class.forName0(Native Method)
	at java.base/java.lang.Class.forName(Class.java:315)
	at org.catacombae.hfsexplorer.MacUtil.registerMacApplicationHandlerInternal(MacUtil.java:93)
	at org.catacombae.hfsexplorer.MacUtil.registerMacApplicationHandler(MacUtil.java:73)
	... 4 more
$
```
This is my Java version:
```
$ java --version
openjdk 11.0.14.1 2022-02-08
OpenJDK Runtime Environment Temurin-11.0.14.1+1 (build 11.0.14.1+1)
OpenJDK 64-Bit Server VM Temurin-11.0.14.1+1 (build 11.0.14.1+1, mixed mode)
$
```

**Comment by unsound** [comment]
This should be fixed already with bffdea7734e22720606a5c2b652d32ec7cb86c9d. Please feel free to reopen this issue if this is not solved in the ```proposed``` branch.

**Comment by elmimmo** [comment]
Please, release a version with the fix. The latest one available, [HFSExplorer 2021.10.9](https://github.com/unsound/hfsexplorer/releases/tag/hfsexplorer-2021.10.9), still produces the error.

**Comment by unsound** [comment]
@elmimmo A new version was released last week, [2026.4.9](https://github.com/unsound/hfsexplorer/releases/tag/hfsexplorer-2026.4.9). Please check that it fixes this issue according to your expectations.

**Comment by elmimmo** [comment]
It does. Now it launches and sees the contents of disk images.
